### PR TITLE
Use Roboto Mono and Stacks

### PR DESCRIPTION
**Pull request by huayunh**
- Use `'Roboto Mono'` whenever possible
- Use `<Stack>` whenever possible

### CODE REVIEW

**Comment by daileytj** [comment]
What's up with all the re-formatting? Was it not prettier before?

**Comment by huayunh** [comment]
> What's up with all the re-formatting? Was it not prettier before?

I don't know. CI didn't pass the check, so I ran `yarn prettier` locally.

> What issue number or bug are you working from here? What prompted the `<Stack>` refactor?

None. `<Stack>` is a cleaner (and perhaps the more encouraged way by MUI) way to handle flex box than grouping everything in a `<Box>`

File: docs/src/index.css [hunk 1]
```diff
@@ -26,7 +26,7 @@ code[class*='language-'],
 pre[class*='language-'],
 code[class*='language-'] *,
 pre[class*='language-'] * {
-    font-family: monospace !important;
+    font-family: 'Roboto Mono' !important;
```
Comment (daileytj) [inline docs/src/index.css:29]: Should we add a monospace fallback here?

File: docs/src/shared/CodeBlock.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ export const CodeBlock: React.FC<CodeBlockProps> = (props): JSX.Element => {
                 data-line={dataLine}
                 style={{ margin: 0, width: '100%', backgroundColor: Colors.black[800], borderRadius: 4 }}
             >
-                <code style={{ fontFamily: 'monospace' }} className={`language-${language}`}>
+                <code style={{ fontFamily: 'Roboto Mono' }} className={`language-${language}`}>
```
Comment (daileytj) [inline docs/src/shared/CodeBlock.tsx:36]: monospace Fallback?

**Review by daileytj — APPROVED** [review]

File: docs/src/shared/CodeBlock.tsx (hunk 2 above)
Comment (daileytj) [reply docs/src/shared/CodeBlock.tsx:36]:
```suggestion
                <code style={{ fontFamily: '"Roboto Mono", monospace' }} className={`language-${language}`}>
```

**Pull request merged**